CHANGES.rst

```diff
@@ -1,4 +1,4 @@
-.. This file should contain the changes for the last release only, which
+[6~].. This file should contain the changes for the last release only, which
```

Comment:
no problem, it's always a good idea to double check your commits  before pushing, but that's what pull requests for as well :-)

I started a jenkins job to test the pull request, let's see if jenkins agrees on your changes.